index.bs

```diff
@@ -1432,8 +1429,131 @@ following additional fields:
 Sessions {#streaming-sessions}
 ------------------------------------
 
-TODO
+To start a streaming session, a sender may send a
+streaming-session-start-request message with the following properties:
 
+: streaming-session-id
+:: Identifies the streaming session.  Must be unique for the (sender,
```

Comment:
The receiver only knows about the IDs that the sender has requested and the receiver has responded to.  The receiver, when demuxing messages, just needs to demux by streaming-session-id.  It doesn't need to be unique across the different senders the receivers receives from (and thus doesn't need a sender-specific prefix).

Reply:
OK, then it could be from the same counter that we use for requests and watches, which is unique per-sender.  Should we just tell implementers to do that?

The receiver internally will need to add the sender fp or some other prefix to demux.

Reply:
It makes sense that we need to do the same thing with forgetting IDs that we do for request IDs, so I'll recommend that.